BMIService.kt

```diff
@@ -0,0 +1,17 @@
+package com.myfitnesstracker.service
+
+import BMI_DAO
+import java.util.*
+import com.myfitnesstracker.ui.dto.BMI
+
+abstract class BMIService: BMI_DAO {
+
+    internal fun fetchUserInputBMI(weight: Double, height: Double){
+
```

Comment:
These methods are incomplete.  Only complete functions should be merged to master.  Otherwise, you're cluttering up the master branch with unused code.